docs: add new stories for UI components

benoitf opened the pull request:
### What does this PR do?
Add new stories for UI components

### Screenshot / video of UI

![image](https://github.com/user-attachments/assets/2f953548-727c-4c4c-8b9d-4b1c34e56373)

https://66f17e753e2f921f3f94916e--podman-desktop-pr.netlify.app/storybook

### What issues does this PR fix or reference?

fixes https://github.com/containers/podman-desktop/issues/7441

### How to test this PR?

<!-- Please explain steps to verify the functionality,
do not forget to provide unit/component tests -->

- [ ] Tests are covering the bug fix or the new feature

axel7083 commented:
The tooltip indication is not really visible in dark mode

![image](https://github.com/user-attachments/assets/d69802b6-5d44-4da9-9001-2f809d6e42c9)

I would remove the next `Hover the icon` and put a `?` icon instead

Otherwise great job, I created https://github.com/containers/podman-desktop/issues/8840 as I just noticed the problem with the numerous items added

feloy reviewed (commented):
- I'm testing by running `pnpm dev` from the `storybook` directory, I'm not sure if there is a better way

- In the Docs page, the Stories are not displaying anything (they are displayed correctly in dedicated pages):

![stories-empty](https://github.com/user-attachments/assets/e2dbd272-02ee-4e54-95a3-45dfaab2c32e)

- In the source code, `<wrapper` is displayed instead of the real component name. 

![tooltip-source](https://github.com/user-attachments/assets/88474d3d-efa2-4175-83ab-c5437115aed4)

- could the terminal displaying the code be in light mode?

benoitf commented:
@feloy I think the best way is to render the final look

or go to the website (click on the netlify link of this PR)

https://66e1a382d9435a070cc59d87--podman-desktop-pr.netlify.app/storybook/

feloy commented:
> @feloy I think the best way is to render the final look
> 
> or go to the website (click on the netlify link of this PR)
> 
> https://66e1a382d9435a070cc59d87--podman-desktop-pr.netlify.app/storybook/

ok, thanks. So the only remaining concern would be:

> could the terminal displaying the code be in light mode?

benoitf commented:
there might be a way but it's not related to this PR that only bring new stories (no terminal being added)

you can create the issue, it'll be for the storybook/docusaurus integration

feloy commented on storybook/src/stories/screen/EmptyScreen.stories.svelte:
```diff
@@ -0,0 +1,46 @@
+<script context="module" lang="ts">
```
Same here, could it be possible to specify title, message, detail, ... in the Docs section?

benoitf replied:
fixed

benoitf commented on storybook/src/stories/ErrorMessage.stories.svelte:
```diff
@@ -0,0 +1,46 @@
+setTemplate(template);
+</script>
+
+{#snippet template({ _children, ...args }: Args<typeof Story>, _context: StoryContext<typeof Story>)}
+  <ErrorMessage {...args} />
+{/snippet}
+
```
it means that the snippet of code that user can see using copy/paste will include that margin

feloy replied:
ok, so let's ignore, thanks

benoitf commented on storybook/src/stories/input/Input.stories.svelte:
```diff
@@ -0,0 +1,64 @@
+<script context="module" lang="ts">
```
fixed

benoitf commented on storybook/src/stories/Tab.stories.svelte:
```diff
@@ -0,0 +1,34 @@
+    'title 1': '',
+    'title 2': '',
+    'title 3': '',
+  }} />
```
fixed

benoitf commented:
I've added a bunch of argsTypes parameters + example on selected for the tab

also added an extra padding on tooltip to see the tooltip

feloy commented on storybook/src/stories/Tab.stories.svelte:
```diff
@@ -0,0 +1,54 @@
+<Story
+  name="Selected"
+  args={{
+    items: [{ title: 'title 1', selected: false }, { title: 'title 2', selected: true }, { title: 'title 3' }],
```
Unfortunately, the `items` value is not displayed in the code and it isnot possible for the user to see what changes. 

Could it be simpler to have a more static template, with let's say 2 tabs, and 4 params title1, selected1, title2 and selected2?

![tab-stories-items](https://github.com/user-attachments/assets/6c51ae4a-b3db-4c81-8c08-402e6a3e3c3b)

benoitf replied:
Here I don't know how to proceed in a good way

As either it's a single tab example 

Or if it's more complex then it's not a single component but multiple components 

I can switch to the single element way

feloy replied:
I suppose the single Tab way would be simpler, and still useful

benoitf replied:
switch to a single Tab

axel7083 reviewed (approved):
LGTM

dgolovin reviewed (commented):
![image](https://github.com/user-attachments/assets/c92a2fc4-be3e-441f-ba52-c681edc0fe8f)

Is it normal for pr preview?

Also noticed that not all the elements have 'Basic' item. Not sure if that is intentional.

feloy reviewed (approved):
LGTM
Thanks for the changes

benoitf commented:
@dgolovin I fixed the dropdown rendering by adding a min-height and a padding

![image](https://github.com/user-attachments/assets/b03ce020-7708-4c59-bf6e-3476609ae8c4)

I renamed also `Example` to `Basic`

dgolovin reviewed (approved):
LGTM now
